src/main/java/com/epam/bookstoreservice/exception/handler/MyExceptionHandler.java

```diff
@@ -6,47 +6,41 @@
 import com.epam.bookstoreservice.exception.BookNotFoundException;
 import com.epam.bookstoreservice.exception.PhoneNumberNotFoundException;
 import com.epam.bookstoreservice.exception.UnmatchedIdException;
-import com.epam.bookstoreservice.exception.WrongPhoneNumberOrPassword;
+import com.epam.bookstoreservice.exception.WrongPhoneNumberOrPasswordException;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
+import javax.servlet.http.HttpServletResponse;
 
+/**
+ * Exception for custom exception
+ */
 @ControllerAdvice
 public class MyExceptionHandler {
-    @ExceptionHandler(value = InsufficientInventoryException.class)
-    public ResponseEntity<Result<String>> dealWithInsufficientInventoryException(InsufficientInventoryException ex) {
-        return ResponseEntity.status(HttpStatus.BAD_REQUEST)
-                .contentType(MediaType.APPLICATION_JSON).body(Result.error(ex.getErrorCode(), ex.getErrorMsg()));
+    @ExceptionHandler({
+            BookNotFoundException.class,
+            PhoneNumberNotFoundException.class
+    })
+    public ResponseEntity<Result<String>> handleNotFoundException(Throwable throwable) {
+        return handlerException(throwable,HttpStatus.NOT_FOUND,HttpServletResponse.SC_NOT_FOUND);
```

Comment:
HttpStatus.NOT_FOUND and HttpServletResponse.SC_NOT_FOUND are the same thing.
You can choose one of them.